modules/cross-account-log-forwarding-troubleshooting.adoc

```diff
@@ -0,0 +1,32 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-10-19
+:_mod-docs-content-type: REFERENCE
+
+[id="cross-account-log-forwarding-troubleshooting_{context}"]
+= Cross-account log forwarding troubleshooting
+
+The following table describes issues that you might encounter with cross-account log forwarding and the steps that you can take to resolve the issues.
+
+.Cross-account log forwarding issue and resolution
+[cols="1,3",options="header"]
+|====
+|Issue
+|Troubleshooting steps
+
+|Role Assumption Fails
+a|
+* Verify the trust relationship in the target account.
+* Verify that the external ID in the trust policy is an exact match with the one used in`ClusterLogForwarder` Custom Resource (CR).
+* Ensure that the initial role has the `sts:AssumeRole` permission.
+
+|Permission Denied
+a|
+* Verify that the assumed role has CloudWatch permissions.
+* Check the CloudWatch resource policies.
+* Verify that region configuration matches.
+
+|Invalid role or user ARN
+a|
+* Ensure that the role Amazon Resource Name (ARN) in the format: `arn:aws:iam::<account>:role/<role-name>`
```

Comment:
arn:<aws-partition>:iam::<account>:role/<role-name>
arn:<aws-partition>:iam::<account>:user/<user-name>
For example:
arn:aws:iam::123456789123:role/role-to-cloudwatch
arn:aws:iam::123456789123:user/<initial-user-when-user-aws-secret-key>
arn:aws-us-gov:iam::123456789123:user/role-to-cloudwatch   when use US Goverment Cloud.